### PR TITLE
Hide local_mbox account

### PR DESCRIPTION
**Pull request by marbetschar**
**Before**

Screenshot from 2021-07-13 06-40-26

**After**

Screenshot from 2021-07-13 06-40-50

### CODE REVIEW

File: src/AccountsModel.vala [hunk 1]
```diff
@@ -59,10 +59,10 @@ public class OnlineAccounts.AccountsModel : Object {
             return;
         }
 
-        // Ignore "Search" and "On This Computer"
+        // Ignore "Search", "On This Computer" and "local_mbox"
         if (e_source.has_extension (E.SOURCE_EXTENSION_MAIL_ACCOUNT)) {
             unowned var mail_source = (E.SourceMailAccount) e_source.get_extension (E.SOURCE_EXTENSION_MAIL_ACCOUNT);
-            if (mail_source.backend_name == "vfolder" || mail_source.backend_name == "maildir") {
+            if (mail_source.backend_name == "vfolder" || mail_source.backend_name == "maildir"  || mail_source.backend_name == "mbox" ) {
```
Comment (JoseExposito) [inline src/AccountsModel.vala:65]:
```suggestion
            if (mail_source.backend_name == "vfolder" || mail_source.backend_name == "maildir" || mail_source.backend_name == "mbox" ) {
```

**Review by JoseExposito — CHANGES_REQUESTED** [review]
Thanks for this PR! Just a little change to make the linter happy.

File: src/AccountsModel.vala (hunk 1 above)
Comment (marbetschar) [reply src/AccountsModel.vala]: Sure thing!

**Comment by JoseExposito** [comment]
@marbetschar thanks, code looks fine, but I'll appreciate a quick test from someone in @elementary/desktop-developers with an account property configured.

**Review by danrabbit — APPROVED** [review]
Works for me :)

**Pull request merged**